DOCS/man/vo.rst

```diff
@@ -667,6 +667,13 @@ Available video output drivers are:
         :yes:   Attempt to enable VRR, whether the capability is reported or not.
         :auto:  Attempt to enable VRR if support is reported.
 
+    ``--drm-send-hdr-meta=<no|auto>``
```

Comment:
this appears to be a "do it or not" switch, so why `auto`?

Reply:
I'm not really sure what is proper for an option like this. If it's yes/no/auto, would you expect "yes" to fail if hdr metadata can't be sent to the screen? Or just a very loud error?

Reply:
Hmm, not sure. Worth checking what the d3d11 hdr option does.

Reply:
d3d option has auto, and then a bunch of other colorspaces, so I think that's consistent with this.